### ISSUE TITLE
Windows tests are broken, version 7.0.3

### ISSUE THREAD

**Issue by alexk-z** [isaacs/node-glob#251]
```
nocase, nomagic
not ok should be equivalent
  +++ found
  --- wanted
  -[
  -  "c:/TMP/A"
  -  "c:/TMP/a"
  -  "c:/tMP/A"
  -  "c:/tMP/a"
  -  "c:/tMp/A"
  -  "c:/tMp/a"
  -  "c:/tmp/A"
  -  "c:/tmp/a"
  -]
  +[]
  at:
    line: 92
    column: 7
    file: test/nocase-nomagic.js
  stack: |
    test/nocase-nomagic.js:92:7
    f (node_modules/once/once.js:17:25)
    Glob.<anonymous> (glob.js:9:6041)
    Glob._finish (glob.js:9:8600)
    done (glob.js:9:7740)
    Glob._processReaddir2 (glob.js:9:19798)
    glob.js:9:17744
    RES (node_modules/inflight/inflight.js:23:14)
    f (node_modules/once/once.js:17:25)
    Glob._readdirEntries (glob.js:9:29460)
    glob.js:9:28301
    FSReqWrap.oncomplete (fs.js:82:15)
  source: |
    t.same(res.sort(), want)

nocase, nomagic
not ok should be equivalent
  +++ found
  --- wanted
  -[
  -  "c:/TMP/A"
  -  "c:/TMP/a"
  -  "c:/tMP/A"
  -  "c:/tMP/a"
  -  "c:/tMp/A"
  -  "c:/tMp/a"
  -  "c:/tmp/A"
  -  "c:/tmp/a"
  -]
  +[]
  at:
    line: 102
    column: 7
    file: test/nocase-nomagic.js
  stack: |
    test/nocase-nomagic.js:102:7
    f (node_modules/once/once.js:17:25)
    Glob.<anonymous> (glob.js:9:6041)
    Glob._finish (glob.js:9:8600)
    done (glob.js:9:7740)
    Glob._processReaddir2 (glob.js:9:19798)
    glob.js:9:17744
    RES (node_modules/inflight/inflight.js:23:14)
    f (node_modules/once/once.js:17:25)
    Glob._readdirEntries (glob.js:9:29460)
    glob.js:9:28301
    FSReqWrap.oncomplete (fs.js:82:15)
  source: |
    t.same(res.sort(), want)

nocase, with some magic
not ok should be equivalent
  +++ found
  --- wanted
   [
  -  "c:/TMP/A"
  -  "c:/TMP/a"
  -  "c:/tMP/A"
  -  "c:/tMP/a"
  -  "c:/tMp/A"
  -  "c:/tMp/a"
  -  "c:/tmp/A"
  -  "c:/tmp/a"
  +  "G:/tmp/glob-test"
   ]
  at:
    line: 128
    column: 7
    file: test/nocase-nomagic.js
  stack: |
    test/nocase-nomagic.js:128:7
    f (node_modules/once/once.js:17:25)
    Glob.<anonymous> (glob.js:9:6041)
    Glob._finish (glob.js:9:8600)
    done (glob.js:9:7740)
    Glob._processReaddir2 (glob.js:9:21305)
    glob.js:9:17744
    RES (node_modules/inflight/inflight.js:23:14)
    f (node_modules/once/once.js:17:25)
    Glob._readdirEntries (glob.js:9:29460)
    glob.js:9:28301
    FSReqWrap.oncomplete (fs.js:82:15)
  source: |
    t.same(res.sort(), want)

nocase, with some magic
not ok should be equivalent
  +++ found
  --- wanted
   [
  -  "c:/TMP/A"
  -  "c:/TMP/a"
  -  "c:/tMP/A"
  -  "c:/tMP/a"
  -  "c:/tMp/A"
  -  "c:/tMp/a"
  -  "c:/tmp/A"
  -  "c:/tmp/a"
  +  "G:/tmp/glob-test"
   ]
  at:
    line: 135
    column: 7
    file: test/nocase-nomagic.js
  stack: |
    test/nocase-nomagic.js:135:7
    f (node_modules/once/once.js:17:25)
    Glob.<anonymous> (glob.js:9:6041)
    Glob._finish (glob.js:9:8600)
    done (glob.js:9:7740)
    Glob._processReaddir2 (glob.js:9:21305)
    glob.js:9:17744
    RES (node_modules/inflight/inflight.js:23:14)
    f (node_modules/once/once.js:17:25)
    Glob._readdirEntries (glob.js:9:29460)
    glob.js:9:28301
    FSReqWrap.oncomplete (fs.js:82:15)
  source: |
    t.same(res.sort(), want)
```

**Comment by isaacs** [comment]
Interesting!  It looks like you'e running it in a working directory in the `G:` drive.  Works for me on `C:`, but it looks like I need to make that test not be drive-specific.

**Comment by isaacs** [comment]
Should be fixed on 68f2509d237babec64cf03adf72531127fd80339

**Comment by alexk-z** [comment]
Different failures on my box:

test/nocase-nomagic.js ................................ 1/5 2s
  nocase, nomagic
  not ok should be equivalent
    +++ found
    --- wanted
     [
    -  "g:/TMP/A"
    -  "g:/TMP/a"
    -  "g:/tMP/A"
    -  "g:/tMP/a"
    -  "g:/tMp/A"
    -  "g:/tMp/a"
    -  "g:/tmp/A"
    -  "g:/tmp/a"
    +  "G:/TMP/A"
    +  "G:/TMP/a"
    +  "G:/tMP/A"
    +  "G:/tMP/a"
    +  "G:/tMp/A"
    +  "G:/tMp/a"
    +  "G:/tmp/A"
    +  "G:/tmp/a"
     ]
    at:
      line: 98
      column: 7
      file: test/nocase-nomagic.js
    stack: |
      test/nocase-nomagic.js:98:7
      f (node_modules/once/once.js:17:25)
      Glob.<anonymous> (glob.js:9:6041)
      Glob._finish (glob.js:9:8600)
      done (glob.js:9:7740)
      Glob._processReaddir2 (glob.js:9:21305)
      glob.js:9:17744
      RES (node_modules/inflight/inflight.js:23:14)
      f (node_modules/once/once.js:17:25)
      Glob._readdirEntries (glob.js:9:29460)
      glob.js:9:28301
      test/nocase-nomagic.js:62:9
    source: |
      t.same(res.sort(), want)

  nocase, nomagic
  not ok should be equivalent
    +++ found
    --- wanted
     [
    -  "g:/TMP/A"
    -  "g:/TMP/a"
    -  "g:/tMP/A"
    -  "g:/tMP/a"
    -  "g:/tMp/A"
    -  "g:/tMp/a"
    -  "g:/tmp/A"
    -  "g:/tmp/a"
    +  "G:/TMP/A"
    +  "G:/TMP/a"
    +  "G:/tMP/A"
    +  "G:/tMP/a"
    +  "G:/tMp/A"
    +  "G:/tMp/a"
    +  "G:/tmp/A"
    +  "G:/tmp/a"
     ]
    at:
      line: 108
      column: 7
      file: test/nocase-nomagic.js
    stack: |
      test/nocase-nomagic.js:108:7
      f (node_modules/once/once.js:17:25)
      Glob.<anonymous> (glob.js:9:6041)
      Glob._finish (glob.js:9:8600)
      done (glob.js:9:7740)
      Glob._processReaddir2 (glob.js:9:21305)
      glob.js:9:17744
      RES (node_modules/inflight/inflight.js:23:14)
      f (node_modules/once/once.js:17:25)
      Glob._readdirEntries (glob.js:9:29460)
      glob.js:9:28301
      test/nocase-nomagic.js:62:9
    source: |
      t.same(res.sort(), want)

  nocase, with some magic
  not ok should be equivalent
    +++ found
    --- wanted
     [
    -  "g:/TMP/A"
    -  "g:/TMP/a"
    -  "g:/tMP/A"
    -  "g:/tMP/a"
    -  "g:/tMp/A"
    -  "g:/tMp/a"
    -  "g:/tmp/A"
    -  "g:/tmp/a"
    +  "G:/TMP/A"
    +  "G:/TMP/a"
    +  "G:/tMP/A"
    +  "G:/tMP/a"
    +  "G:/tMp/A"
    +  "G:/tMp/a"
    +  "G:/tmp/A"
    +  "G:/tmp/a"
     ]
    at:
      line: 134
      column: 7
      file: test/nocase-nomagic.js
    stack: |
      test/nocase-nomagic.js:134:7
      f (node_modules/once/once.js:17:25)
      Glob.<anonymous> (glob.js:9:6041)
      Glob._finish (glob.js:9:8600)
      done (glob.js:9:7740)
      Glob._processReaddir2 (glob.js:9:21305)
      glob.js:9:17744
      RES (node_modules/inflight/inflight.js:23:14)
      f (node_modules/once/once.js:17:25)
      Glob._readdirEntries (glob.js:9:29460)
      glob.js:9:28301
      test/nocase-nomagic.js:62:9
    source: |
      t.same(res.sort(), want)

  nocase, with some magic
  not ok should be equivalent
    +++ found
    --- wanted
     [
    -  "g:/TMP/A"
    -  "g:/TMP/a"
    -  "g:/tMP/A"
    -  "g:/tMP/a"
    -  "g:/tMp/A"
    -  "g:/tMp/a"
    -  "g:/tmp/A"
    -  "g:/tmp/a"
    +  "G:/TMP/A"
    +  "G:/TMP/a"
    +  "G:/tMP/A"
    +  "G:/tMP/a"
    +  "G:/tMp/A"
    +  "G:/tMp/a"
    +  "G:/tmp/A"
    +  "G:/tmp/a"
     ]
    at:
      line: 141
      column: 7
      file: test/nocase-nomagic.js
    stack: |
      test/nocase-nomagic.js:141:7
      f (node_modules/once/once.js:17:25)
      Glob.<anonymous> (glob.js:9:6041)
      Glob._finish (glob.js:9:8600)
      done (glob.js:9:7740)
      Glob._processReaddir2 (glob.js:9:21305)
      glob.js:9:17744
      RES (node_modules/inflight/inflight.js:23:14)
      f (node_modules/once/once.js:17:25)
      Glob._readdirEntries (glob.js:9:29460)
      glob.js:9:28301
      test/nocase-nomagic.js:62:9
    source: |
      t.same(res.sort(), want)
